### PR TITLE
Ap 3678 remove passported permissions

### PR DESCRIPTION
**Pull request by kmahern**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3678)

Removed seeds for passported and non-passported permissions, as these are now both defaults.
Updated code and tests to remove dependencies on these permissions.
Removed redundant use_ccms_arbiter class.
Removed redundant permissions_updater task helper class.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/providers/check_benefits/index.html.erb [hunk 1]
```diff
@@ -1,5 +1 @@
-<% if should_use_ccms? %>
-  <%= render 'use_ccms' %>
```
Comment (naseberry) [inline app/views/providers/check_benefits/index.html.erb:2]: By removing the redundant `use_ccms_arbiter` service class is the `_use_ccms.html.erb` partial also redundant?

File: app/views/providers/check_benefits/index.html.erb [hunk 2]
```diff
@@ -1,5 +1 @@
...
-<% else %>
-  <%= render 'check_benefits_result' %>
-<% end %>
+<%= render 'check_benefits_result' %>
```
Comment (cpjmcquillan) [inline app/views/providers/check_benefits/index.html.erb:1]: Can we remove the partial and move the content into this template?

As far as I can tell the partial isn't used anywhere else.

File: spec/requests/admin/roles/permissions_spec.rb [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
     it "shows the correct permissions" do
       subject
-      expect(response.body).to include("application.passported._")
+      expect(response.body).not_to include("application.passported._")
```
Comment (cpjmcquillan) [inline spec/requests/admin/roles/permissions_spec.rb:24]: Does this test make sense still now that passported permissions have been removed?

**Review by cpjmcquillan — APPROVED** [review]
🧹

File: spec/requests/admin/roles/permissions_spec.rb (hunk 3 above)
Comment (kmahern) [reply spec/requests/admin/roles/permissions_spec.rb]: 👍 have removed this test

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

File: spec/requests/providers/check_benefits_spec.rb [hunk 4]
```diff
@@ -218,7 +218,7 @@
       let(:application) { create(:legal_aid_application, :with_positive_benefit_check_result, :checking_applicant_details, applicant:, provider:) }
 
       context "with permissions passported" do
-        let(:provider) { create(:provider, :with_passported_permissions) }
```
Comment (jsugarman) [inline spec/requests/providers/check_benefits_spec.rb:221]: expanding this out i notice there are now two tests, doing the same thing.

Comment (jsugarman) [reply spec/requests/providers/check_benefits_spec.rb:221]: the `context`s at lines 220 and 229 can probably be removed and replaced with a single `it` example

Comment (jsugarman) [reply spec/requests/providers/check_benefits_spec.rb:221]:
```
        it "displays positive DWP result content" do
          get "/providers/applications/#{application.id}/check_benefits"
          expect(response.body).to include("DWP records show that your client receives a passporting benefit")
        end
```

**Review by jsugarman — APPROVED** [review]
💯 LGTM

 possibly a few repeated spec still lurking, like the one i commented but looks great other than that

**Comment by jsugarman** [comment]
@kmahern from the sentry alerts on UAT I am thinking that [this alert](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/app/models/provider.rb#L36) will no longer need to be captured. I guess on production the firms will still have the passported and non-passported permissions in any event and so would not be raised there but that raise the question of whether we are going to have a follow up PR or bit of work to remove existing firm permissions that is no longer required.

File: spec/requests/providers/check_benefits_spec.rb (hunk 4 above)
Comment (kmahern) [reply spec/requests/providers/check_benefits_spec.rb:221]: 👍 have replaced with single test as suggested.

**Comment by kmahern** [comment]
> @kmahern from the sentry alerts on UAT ([example](https://sentry.io/organizations/ministryofjustice/issues/3811658205/events/9f1e4dd8dbfb49efafdb7134a4661623/?project=5416052)) I am thinking that [this alert](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/app/models/provider.rb#L36) will no longer need to be captured. I guess on production the firms will still have the passported and non-passported permissions in any event and so would not be raised there but that raises the question of whether we are going to have a follow up PR or bit of work to remove existing firm permissions that is no longer required.

@jsugarman thanks for spotting this, I have removed the alert. FYI @colinbruce has created a tidy function in db/seeds/permissions.rb as part of an earlier pr that will remove unused permissions from providers - which is why it was alerting in UAT as the default is now that a provider will have no permissions.

**Pull request merged**